README.md

```diff
@@ -126,6 +128,15 @@ const fsClient = init({ apiKey: '<YOUR_API_KEY>' });
     });
   ```
 
+#### Per-Request Options
+- Use `withRequestOptions` to apply options for a single request.
+
+  ```ts
+  const createResponse = await users
+      .withRequestOptions({ idempotencyKey: 'YOUR_KEY' })
```

Comment:
I like being able to set options like this, very clean.

Reply:
Yeah I kinda went back and forth with weather to introduce `withRequestOptions` or just have the alternate option where the request object like `users.update(request: {...}, { idempotencyKey: 'YOUR_KEY' })`..

Reply:
I guess we could do both: users.withRequestOptions( { idempotencyKey: 'YOUR_KEY' } ).update({id:string, body:Request, include_schema: bool})..